### PR TITLE
Jones DAO - updating tvl with STIP

### PR DESCRIPTION
**Pull request by 0xIvan**
**NOTE**

> - If you would like to add a `volume` adapter please submit the PR [here](https://github.com/DefiLlama/adapters).
> - If you would like to add a `liquidations` adapter, please refer to [this readme document](https://github.com/DefiLlama/DefiLlama-Adapters/tree/main/liquidations) for details.

1. Once your adapter has been merged, it takes time to show on the UI. If more than 24 hours have passed, please let us know in Discord.
2. Please enable "Allow edits by maintainers" while putting up the PR.
3. Sorry, We no longer accept fetch adapter for new projects, we prefer the tvl to computed from blockchain data, if you have trouble with creating a the adapter, please hop onto our discord, we are happy to assist you.
4. Please fill the form below  **only if the PR is for listing a new protocol** else it can be ignored/replaced with reason/details about the PR
5. **For updating listing info** It is a different repo, you can find your listing in this file: https://github.com/DefiLlama/defillama-server/blob/master/defi/src/protocols/data2.ts, you can  edit it there and put up a PR
6. Do not edit/push `package-lock.json` file as part of your changes, we use lockfileVersion 2, and most use v1 and using that messes up our CI
7. No need to go to our discord and announce that you've created a PR, we monitor all PRs and will review it asap

---
##### Github org/user (Optional, if your code is open source, we can track activity):

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/jones-dao exports TVL: 
        
 
 
    ethereum                  2.47 M
    arbitrum-pool2            229.82 k
    arbitrum-staking          5.08 M
    arbitrum                  16.46 M
    pool2                     229.82 k
    staking                   5.08 M
    
    total                    18.93 M

File: projects/jones-dao/index.js [hunk 1]
```diff
@@ -1,89 +1,100 @@
...
-      }
-      return `arbitrum:${addr}`;
+      return `arbitrum:${jAssetToAsset[addr] ?? addr}`;
     }),
-    staking: stakings(addresses.jonesStaking, addresses.jones)
+    staking: stakings(
+      addresses.stakingContracts,
+      [
+        addresses.tokens.jones,
+        addresses.tokens.jglp,
```
Comment (g1nt0ki) [inline projects/jones-dao/index.js:88]: hi @0xIvan 
thanks for the PR, it is mostly good, but isnt the back backing of jglp, jusdc and wjaura already counted as part of tvl?

Comment (g1nt0ki) [reply projects/jones-dao/index.js:88]: if so, counting it under staking would be doublecounting

**Comment by g1nt0ki** [comment]
closing this as there was no response

**Comment by 0xIvan** [comment]
> closing this as there was no response

Hi sorry, I didn't receive a notification of your previous comment. Will address it.

**Comment by llamatester** [comment]
The adapter at projects/jones-dao exports TVL: 
        
 
 
    ethereum                  2.35 M
    arbitrum-pool2            232.88 k
    arbitrum-staking          16.62 M
    arbitrum                  25.43 M
    pool2                     232.88 k
    staking                   16.62 M
    
    total                    27.77 M

**Comment by llamatester** [comment]
The adapter at projects/jones-dao exports TVL: 
        
 
 
    arbitrum-staking          3.33 M
    arbitrum-pool2            232.88 k
    ethereum                  2.35 M
    arbitrum                  25.43 M
    staking                   3.33 M
    pool2                     232.88 k
    
    total                    27.77 M

**Pull request merged**